gradle-jdks/src/main/java/com/palantir/gradle/jdks/PalantirCaPlugin.java

```diff
@@ -149,13 +150,36 @@ private static Optional<String> selectPalantirCertificate(byte[] multipleCertifi
                 .map(PalantirCaPlugin::encodeCertificate);
     }
 
-    private static Collection<? extends Certificate> parseCerts(byte[] multipleCertificateBytes) {
+    // visible for testing
```

Comment:
just use the annotation?

Reply:
I'm not going to bring in Guava for just that

Reply:
Oh wait, this uses guava. Sad :(